sky/resources.py

```diff
@@ -2075,6 +2118,12 @@ def __setstate__(self, state):
         if version < 24:
             self._volumes = None
 
+        if version < 25:
+            original_network_tier = state.get('_network_tier', None)
+            if original_network_tier is not None:
+                state['_network_tier'] = resources_utils.NetworkTier(
```

Comment:
minor, seems that we don't need to reconstruct the `NetworkTier`, as the argument of the `__init__` class should handle that already.

Reply:
Updated to reflect state